### PR TITLE
[backend] Don't try to use uninitialized variable in error message.

### PR DESCRIPTION
**Pull request by trhd**

### CODE REVIEW

File: src/api/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -137,7 +137,7 @@ def extract_proxy_user
       unless @http_user
         if ::Configuration.registration == "deny"
           logger.debug("No user found in database, creation disabled")
-          raise AuthenticationRequiredError.new "User '#{login}' does not exist<br>#{errstr}"
+          raise AuthenticationRequiredError.new "User '#{login}' does not exist<br>"
```
Comment (bgeuken) [inline src/api/app/controllers/application_controller.rb]: Please also remove the <br> element

**Comment by bgeuken** [comment]
@trhd Thank you for the patch! One nitpick, please change the [backend] tag in the commit message to [api]. app/controllers/application_controller.rb is only used by the API.

**Comment by hennevogel** [comment]
Duplicate of #2361

**Comment by hennevogel** [comment]
argh the other way around...

**Comment by trhd** [comment]
Updated the patch.

Wasn't quite sure whether the error message should end in a period or not...

**Comment by hennevogel** [comment]
LGTM. Thank you @trhd 💐

**Pull request merged**